.github/workflows/elixir.yaml

```diff
@@ -0,0 +1,114 @@
+name: Continuous Integration (Elixir)
+
+on:
+  push:
+    branches: main
+    paths:
+      - 'api/**'
+      - '.github/workflows/elixir.yaml'
+  pull_request:
+    paths:
+      - 'api/**'
+      - '.github/workflows/elixir.yaml'
+
+concurrency:
+  group: ci-${{ github.ref }}
+  cancel-in-progress: true
```

Comment:
what does this block in the workflow do?

Reply:
It limits the workflow to one concurrent run per branch. If an update to the branch is pushed while CI is running, the in-progress CI run is cancelled.